frontends/election-manager/src/app.test.tsx

```diff
@@ -148,36 +151,10 @@ afterEach(() => {
 });
 
 async function authenticateWithAdminCard(card: MemoryCard) {
-  // Machine should be locked
-  await screen.findByText('VxAdmin is Locked');
-  card.insertCard(
-    makeAdminCard(eitherNeitherElectionDefinition.electionHash, '123456')
+  return authenticateWithAdminCardNoDefinition(
```

Comment:
Interesting - I see we're wrapping in this way so that the changes to this test file are minimal.

That said, renaming `authenticateWithAdminCard` from `../test/util` `authenticateWithAdminCardNoDefinition` could be confusing since that function does require and use an election definition.

A clearer alternative might be to have the util function take an optional election definition that defaults to `eitherNeitherElectionDefinition`:

```ts
export async function authenticateWithAdminCard(
  card: MemoryCard,
  electionDefinition: ElectionDefinition = eitherNeitherElectionDefinition
): Promise<void>
```

Then, no wrapping should be necessary in this file

Reply:
I ended up just using the helper with no default. I think for people using this helper in the future, it will be easier to force them to specify an election definition rather than risk potential confusion of why their tests are failing due to the wrong card.

Reply:
Also the real reason is I have a vendetta against the `eitherNeitherElectionDefinition` :)

Reply:
LOL that makes a lot of sense to me! To always have an explicit specification